sql/python/sqlflow_submitter/db.py

```diff
@@ -180,7 +180,9 @@ def reader():
             if driver == "mysql":
                 conn.ping(True)
             for row in rows:
-                label = row[label_idx] if label_idx is not None else None
+                # NOTE: If there is no label clause in the extened SQL, the default label value would 
+                # be -1, the Model implementation can determine use it or not.
+                label = row[label_idx] if label_idx is not None else -1
```

Comment:
I am worried that these two situations cannot be distinguished: 1) `row[label_idx] = -1`; 2) `else -1`

Reply:
You're right, `-1` can not be used to distinguish `with label clause` and `without label clause`, the model implementation should deal with that.